src/ios/orb.yml

```diff
@@ -233,6 +233,8 @@ jobs:
                   else
                     mkdir -p ~/.ssh
 
+                    (umask 077; touch ~/.ssh/pod_repo_push_deploy_key.pub)
```

Comment:
Is this line required if we move the `chmod 600` after the key is created?

Reply:
I actually wondered about this too. But this is how CircleCI's official `checkout` step creates the `id_rsa` key from the `$CHECKOUT_KEY` env var, so I thought it safer to do the same steps than the ones they do here, in case I was missing some edge case…